aspnetcore/publishing/index.md

```diff
@@ -12,22 +12,40 @@ ms.technology: aspnet
 ms.prod: asp.net-core
 uid: publishing/index
 ---
-# Hosting and deployment of ASP.NET Core apps
-
-The following articles are available for learning about how to set up hosting environments and deploy ASP.NET Core applications.
-
-* [Host on Windows with IIS](xref:publishing/iis)
-  * [ASP.NET Core Module Configuration Reference](xref:hosting/aspnet-core-module)
-  * [IIS Modules with ASP.NET Core](xref:hosting/iis-modules)
-* [Host in a Windows service](xref:hosting/windows-service)
-* [Host on Linux with Nginx](xref:publishing/linuxproduction)
-* [Host on Linux with Apache](xref:publishing/apache-proxy)
-* [Host in Docker](xref:publishing/docker)
-  * [Building Docker Images](https://docs.microsoft.com/dotnet/articles/core/docker/building-net-docker-images)
-  * [Visual Studio Tools for Docker](https://docs.microsoft.com/dotnet/articles/core/docker/visual-studio-tools-for-docker)
-  * [Publish to a Docker Image](https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker)
-* [Publish to Azure with Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs)
-* [Continuous deployment to Azure with Visual Studio and Git](xref:publishing/azure-continuous-deployment)
-* [Continuous deployment to Azure with VSTS](xref:publishing/vsts-continuous-deployment)
-* [Publish profiles in Visual Studio](xref:publishing/web-publishing-vs)
-* [Directory structure](xref:hosting/directory-structure)
+# Hosting and deployment overview for ASP.NET Core apps
+
+Here are the main steps you perform to deploy an ASP.NET Core app to a hosting environment:
+
+* Publish the app's runtime files to a folder, and copy the folder to a server.
```

Comment:
Do you want to say
**Publish the app to a folder on the deployment server.**
runtime files doesn't tell you much. The publish tool figures out what's needed. If they want to publish local and copy they can, but they can publish directly to the deployment server without an explict copy step.